src/osd/PG.cc

```diff
@@ -3847,6 +3850,30 @@ void PG::replica_scrub(
     return;
   }
 
+  scrubber.active_rep_scrub = op;
+  scrubber.map_state = PG::Scrubber::REG_SCRUBBING;
+  osd->queue_for_build_scrub_map(this, g_conf->osd_scrub_map_start_cost);
+}
+
+void PG::replica_scrub(epoch_t queued, ThreadPool::TPHandle &handle)
```

Comment:
The name of this method should probably be something more like do_scrubmap_work (it runs on the primary as well, right?)